app/assets/javascripts/actions/story.js

```diff
@@ -171,3 +191,13 @@ export const deleteStory = (storyId, projectId) =>
       return dispatch(storyFailure(storyId, error))
     }
   }
+
+export const highlight = storyId =>
+  async (dispatch, getState, {}) => {
+    try {
+      dispatch(highlightStory(storyId, true));
+      setTimeout(() => dispatch(highlightStory(storyId, false)), 400);
```

Comment:
What do you think about renaming this action? Calling it with `highlightStory(id, boolean)` makes it a little unclear.

What do you think: `updateHighlight(storyId, { highlighted: true })` (or `on` instead of `highlighted`).